tests/test_PoloniexAPI.py

```diff
@@ -0,0 +1,47 @@
+import time
+
+# Hack to get relative imports - probably need to fix the dir structure instead but we need this at the minute for
+# pytest to work
+import os, sys, inspect
```

Comment:
@ckcollab currently there are only 2 test files, it would be great if you can fix it. :)